CHANGES.rst

```diff
@@ -66,9 +68,6 @@ Cubeviz
 - Fixed a bug where having Subset breaks coordinates information display
   in image viewers. [#1472]
 
-Other Changes and Additions
```

Comment:
Why is this removed? Is it on purpose?

Reply:
Nope, not sure how that got in there; I'll revert that